plugins/poll/test/javascripts/component/poll-buttons-dropdown-test.js

```diff
@@ -46,6 +46,42 @@ module("Poll | Component | poll-buttons-dropdown", function (hooks) {
     );
   });
 
+  test("Renders a show-tally button when poll is a bar chart", async function (assert) {
+    this.setProperties({
+      closed: false,
+      voters: 2,
+      isStaff: false,
+      isMe: false,
+      topicArchived: false,
+      groupableUserFields: ["stuff"],
+      isAutomaticallyClosed: false,
+      dropDownClick: () => {},
+      availableDisplayMode: "showTally",
+    });
+
+    await render(hbs`<PollButtonsDropdown
+      @closed={{this.closed}}
+      @voters={{this.voters}}
+      @isStaff={{this.isStaff}}
+      @isMe={{this.isMe}}
+      @topicArchived={{this.topicArchived}}
+      @groupableUserFields={{this.groupableUserFields}}
+      @isAutomaticallyClosed={{this.isAutomaticallyClosed}}
+      @dropDownClick={{this.dropDownClick}}
+      @availableDisplayMode={{this.availableDisplayMode}}
+    />`);
+
+    await click(".widget-dropdown-header");
+
+    assert.strictEqual(count("li.dropdown-menu__item"), 2);
+
+    assert.strictEqual(
+      query("li.dropdown-menu__item span").textContent.trim(),
+      I18n.t("poll.show-tally.label"),
+      "displays the show absolute button"
+    );
```

Comment:
query dom :p sorry Im nitpicking with this, but it's good to take the habits to use them instead, so much more expressive

Reply:
done!